packages/gatsby-plugin-sharp/src/index.js

```diff
@@ -144,7 +144,11 @@ async function generateBase64({ file, args, reporter }) {
   })
   let pipeline
   try {
-    pipeline = sharp(file.absolutePath).rotate()
+    pipeline = sharp(file.absolutePath)
+
+    if (!options.rotate || options.rotate === 0) {
```

Comment:
0 is a falsy value so it should be covered by `!options.rotate`
```suggestion
    if (!options.rotate) {
```